CONTRIBUTING.md

```diff
@@ -48,7 +48,7 @@ encourage all to read it carefully.
 
 Discussion often happens in the issue tracker and in pull requests.
 In addition, there is
-[Gitter chat room](https://gitter.im/GenericMappingTools/gmtsphinxext)
+[Gitter chat room](https://gitter.im/GenericMappingTools/sphinx_gmt)
```

Comment:
We don't have this chatroom.

Reply:
I don't think we need a separate one for this. You could probably remove that part. Or we could use https://gitter.im/GenericMappingTools/Lobby but there is very little traffic in any of them.